### PR TITLE
Cloud 2002.0.15 release

### PR DESCRIPTION
**Pull request by hguthrie**
This contains the updates for the Cloud 2002.0.15 release.

- [x] waiting on additional fixes
- [x] waiting to merge the Global variable fixes
- [x] dev review
- [x] whatsnew updated with links
- [x] editor review
- [x] link check

whatsnew
Release notes for Cloud ece-tools version [2002.0.15](https://devdocs.magento.com/guides/v2.2/cloud/release-notes/cloud-tools.html#v2002015)
Added a new [Cron container](
https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-development.html#cron-container) to the Cloud Docker architecture.
Added a [services table](https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-config.html) to clarify the service key and version values available for the Cloud Docker environment setup.
There are several other Cloud Docker updates, such as a new a PHP 7.2 image and various services, the ability to [share data between host and container](
https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-development.html#sharing-data-between-host-machine-and-container),  and information on [importing a database dump](
https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-development.html#database-container).
Added a [Quick Reference](https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-quick-reference.html) for the Docker environment.
Added the [ENABLE_GOOGLE_ANALYTICS environment variable](https://devdocs.magento.com/guides/v2.2/cloud/env/variables-deploy.html#enable-google-analytics).
Moved the [Global environment variables](https://devdocs.magento.com/guides/v2.2/cloud/env/variables-global.html) to a new, independent topic.

### CODE REVIEW

File: guides/v2.2/cloud/release-notes/cloud-tools.md [hunk 1]
```diff
@@ -22,7 +22,41 @@ php ./vendor/bin/ece-tools list
...
+
+    -  <!-- MAGECLOUD-2565 -->Added a [Cron container]({{page.baseurl}}/cloud/docker/docker-development.html#cron-container) based on the PHP-CLI image.
+
+    -  Added the following services to the Docker build:
+
+        -  <!-- MAGECLOUD-2567 & 2889-->RabbitMQ 3.5 and 3.7
+        -  <!-- MAGECLOUD-2569 & 2887 -->ElasticSearch 1.7 and 2.4
+        -  <!-- MAGECLOUD-2886 -->Redis 3.2 and 4.0
+
+    -  <!-- MAGECLOUD-2577 -->Now you have the DB dump capability when using Cloud Docker [database container]({{page.baseurl}}/cloud/docker/docker-development.html#database-container). Also, you can [share files]({{page.baseurl}}/cloud/docker/docker-development.html#sharing-data-between-host-machine-and-container) between a host machine and a container using the `docker/mnt` directory.
```
Comment (meker12) [inline guides/v2.2/cloud/release-notes/cloud-tools.md]: using "the" Cloud Docker [database container]

File: guides/v2.2/cloud/release-notes/cloud-tools.md [hunk 2]
```diff
@@ -22,7 +22,41 @@ php ./vendor/bin/ece-tools list
...
+
+-  <!-- MAGECLOUD- 2575 -->**Configure with PHP constants**—Added support for [PHP constants]({{page.baseurl}}/cloud/project/magento-env-yaml.html#php-constants) in the `.magento.env.yaml` configuration file.
+
+-  <!--MAGECLOUD-2879-->**New Environment variable**—By default, only the Production environment has Google Analytics enabled. You can enable Google Analytics on the Staging and Integration environments using the  [ENABLE_GOOGLE_ANALYTICS]({{page.baseurl}}/cloud/env/variables-deploy.html#enable_google_analytics) environment variable.
+
+#### Resolved Issues
+
+-  <!-- MAGECLOUD-2862 -->Fixed an issue involving cron processes that prevented the start of the post-deploy phase, when enabled. Now, if you have the post-deploy hook enabled, the cron processes are enabled again at the beginning of the post-deploy phase.
```
Comment (meker12) [inline guides/v2.2/cloud/release-notes/cloud-tools.md]: not sure what "when enabled" refers to .... please check

File: guides/v2.1/cloud/env/variables-global.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+group: cloud-guide
+title: Global variables
+functional_areas:
+  - Cloud
+  - Configuration
+---
+
+The following _global_ variables control actions in the build, deploy, and post-deploy stages of the `.magento.env.yaml` file. Because global variables impact every stage, you must set them in the `global` stage. Insert these variables in the `global` stage of the `.magento.env.yaml` file:
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.1/cloud/env/variables-global.md]: I cleaned this a little bit, but the parent topic explains stage. And each child topic **should** say "control actions in phase" and "set var in stage", roughly.   Stage is a section of the ENV.YAML file. Phase references time. Configurations in the stage section happen during the phase. So, in this case, the configuration would be in `stage > global > key:value`.

File: guides/v2.1/cloud/docker/docker-development.md [hunk 4]
```diff
@@ -41,27 +45,36 @@ The CLI container is based on a 7.0-cli or 7.1-cli image, provides `magento-clou
...
+
+#### Importing a database dump
+
+To import a database dump, place the SQL file into the `docker/mysql/docker-entrypoint-initdb.d` folder. The `{{site.data.var.ct}}` package imports and processes the SQL file the next time you build and start the Docker environment using the `docker-compose up` command.
+
+Although it is a more complex approach, you can use GZIP by _sharing_ the `.sql.gz` file using the `docker/mnt` directory and importing it inside the Docker container.
+
+## Sharing data between host machine and container
+
+You can share files easily between your machine and a Docker container by placing the files in the `docker/mnt` directory. The files appear in `/mnt` directory the next time you build and start the Docker environment using the `docker-compose up` command.
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.1/cloud/docker/docker-development.md]: ✨✨

File: guides/v2.2/cloud/release-notes/cloud-tools.md (hunk 2 above)
Comment (hguthrie) [reply guides/v2.2/cloud/release-notes/cloud-tools.md]: You make a good point. I think I have been too broad with that, and not just here. That phase is not available to every project. Rethinking...

**Comment by dshevtsov** [comment]
@hguthrie whatsnew had new lines inside hyperlinks in several cases.
I removed them to make the whatsnew automation work.

**Pull request merged**